packages/core-js-compat/src/data.js

```diff
@@ -787,6 +787,13 @@ module.exports = {
     firefox: '37',
     safari: '9.0',
   },
+  'es.regexp.sticky': {
+    chrome: '49',
+    edge: '13',
+    firefox: '3',
+    opera: '36',
+    safari: '10.0',
```

Comment:
I got this data from MDN. Note that on Firefox 3 the `y` flags doesn't correctly work with regexp methods, but this compat entry is only about the `.sticky` accessor.

Reply:
We don't need data for modern `opera` - it's generated from `chrome`.